src/main/java/cuchaz/enigma/utils/LangUtils.java

```diff
@@ -0,0 +1,74 @@
+package cuchaz.enigma.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Map;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+
+import cuchaz.enigma.config.Config;
+
+public class LangUtils {
+	public static final String DEFAULT_LANGUAGE = "en_us";
+	private static final Gson GSON = new Gson();
+	
+	public static String translate(String key) {
+		return translate(key, Config.getInstance().language);
+	}
+	
+	public static String translate(String key, String language) {
```

Comment:
Can we cache the language content so we don't do file io every time we look up a key